library/packages/src/lib/packages/update_messages_view.rb

```diff
@@ -0,0 +1,36 @@
+require "yast"
+
+module Packages
+  # This class converts a set of commit results into string representations.
+  #
+  # At this time, richtext is the only provided conversion.
+  class UpdateMessagesView
+    include Yast::I18n
+    extend Yast::I18n
+
+    def initialize(commit_results)
+      textdomain "base"
+      @commit_results = commit_results
+    end
+
+    # Convert a list of messages into richtext
+    #
+    # @param messages [Array<UpdateMessage>] List of messages
+    # @return [String] Richtext representation of the list of messages
+    def richtext
+      text = "<h1>#{_("Packages messages")}</h1>"
+      text << @commit_results.map { |m| message_to_richtext(m) }.join("<hr>")
+    end
+
+  private
+
+    # Convert one message to richtext
+    #
+    # @return [String] Message converted to richtext
+    def message_to_richtext(message)
+      location = format(_("This message will be available at %s"), message.installation_path)
+      "<h2>#{message.solvable}</h2><p><em>#{location}</em></p>" \
+        "<br>#{message.text.strip.gsub("\n", "<br>")}"
```

Comment:
NP: To be on the safe side you should also escape the solvable name and the file location. 

Just a note: Alternatively you could use an ERB template file. Then you could simply use the usual `h()` ERB shortcut for escaping, see this [registration example](https://github.com/yast/yast-registration/blob/master/src/data/registration/certificate_summary.erb).

But for such a short message an inline string is still acceptable. Just a hint if you would extend the message or formatting later... :wink: